pkg/solana/assets/vote-account.sh

```diff
@@ -0,0 +1,49 @@
+# -*- mode: shell-script -*-
+# shellcheck shell=bash
+
+vote-account-create () {
+    local identity_keypair vote_account_keypair auth_withdrawer_keypair args
+
+    identity_keypair=$(temp::file)
```

Comment:
These are within the runner dir?

Reply:
They're not. They're in the running script's temporary directory: 

https://github.com/abklabs/svmkit/blob/main/pkg/runner/assets/lib.bash#L44

We've got an issue to do some file permission auditing.